### DISCUSSION TITLE
Welcome to Anime-batch-downloader-gui Discussions!

### DISCUSSION THREAD

**Discussion by TheYogMehta** [Incredibleflamer/Anime-batch-downloader-gui#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kersplash-wav** [comment]
How do I get subtitles for subbed? I can't find any subtitles files

**Comment by TheYogMehta** [comment]
go in settings and select Anime Provider as zoro, after doing that you will have one more option Merge English Subtitles make sure its OFF

**Reply by kersplash-wav** [reply]
Thanks!I had that option on.

**Comment by kersplash-wav** [comment]
Could we have an option to choose the subtitle file type? Right now it's .ttv but I'd like it to be .srt since windows media player can't use .ttv

**Reply by TheYogMehta** [reply]
will add an option soon

**Reply by TheYogMehta** [reply]
Hey @ECX-0, The latest version now supports changing subtitle formats between **TTV** and **SRT**!

**Note**: This option will only appear if:
- **Provider** is set to **Zoro**
- **Merge Subtitles** is **off**

Enjoy the new feature! 😊

**Reply by kersplash-wav** [reply]
Thank you so much!!!

**Comment by Matt2o24** [comment]
Could we have an option to choose how the files are named E.g. S01E01? Would make downloading straight to Plex a lot easier.

**Reply by rifatozkancomtr** [reply]
`\\${seriesTitle}\\Season ${season}\\${seriesTitle} - S${season}E${episode}`

Plex için de benzer bir isimlendirme gayet iyi olurdu.

**Reply by TheYogMehta** [reply]
sure, i can add it!

**Reply by Matt2o24** [reply]
Amazing! Thank you!

**Comment by Yokubariii** [comment]
Its been a while since i used this, I love the new UI! Thank you for your continuous updates.

**Reply by TheYogMehta** [reply]
thank you 🥰

**Comment by dragonarch0** [comment]
How does local anime/manga work? When I download anime or manga, it doesn't appear there? Am I missing something?

**Comment by ketankumarsinha160-dot** [comment]
hey i new i want to start downloading but i can't even start it , can you help me from the start

**Comment by TheYogMehta** [comment]
ermmm collage exams and final year so cant focus on this project let me get a little free and imma fix everything just 3-4 weeks more 😿

**Reply by TheYogMehta** [reply]
extremely sorry about no update from so many months 🙇‍♂️

**Comment by Daisukanama** [comment]
Can we add hentai to be downloaded? :3
